core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/GenericExecParser.scala

```diff
@@ -59,7 +59,7 @@ class GenericExecParser(
   }
 
   protected def getExprString: String = {
-    node.desc.replaceFirst(s"${node.name} ", "")
+    node.desc.replaceFirst(s"${node.name}\\s*", "")
```

Comment:
Yeah I remember seeing `HashAggregate(` in the description
QQ: Is it possible that the `node.name` may not come at the beginning of the node description?
In that case we better use a regex `^(node_name)\\s*` to guarantee that we replace node_name when it is at the beginning of the description.

Reply:
Thanks @amahussein ! Updated it. Adding of expression parsers was missed during refactoring. Since we are getting the expressions in https://github.com/NVIDIA/spark-rapids-tools/pull/1431, we should be able to update some of the tests to capture these kinds of bugs.

Reply:
Thanks @nartal1 !